src/Content/AttachmentLocal.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Phlib\Mail\Content;
+
+use Phlib\Mail\Exception\RuntimeException;
+
+/**
+ * Attachment class used to add local files to a Mail object
+ *
+ * @package Phlib\Mail
+ */
+class AttachmentLocal extends Attachment
```

Comment:
How about `AttachmentFile` or `AttachmentFromFile`, and making the *filename* a constructor parameter, so immutable?

This class is designed to help a developer create a new part that matches the `Attachment` "interface" (so extends `Attachment`) from a file on their local filesystem. Essentially it's a shortcut to:

```php
$part = new Attachment(mime_content_type($filename));
$part->setContent(file_get_contents($filename));
$part->setName(basename($filename));
```